packages/web/localizations/en.json

```diff
@@ -37,7 +37,7 @@
     "myPools": "My Pools",
     "APR": "APR",
     "available": "Available",
-    "liquidity": "Pool Liquidity",
```

Comment:
This caused Frontend to display the pool's total liquidity as 'My liquidity', which is incorrect